tests/test_main.py

```diff
@@ -1,20 +1,21 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 import sys
-from runpy import run_module
 from unittest.mock import patch
 
+from pydantic import ValidationError
 from pytest import mark, raises
 
+from retasc.__main__ import main
+
 
 def run_main(*args, code=None):
```

Comment:
Can you change the `code` argument to `expected_exit_code` or something clearer?